src/main/java/domain/OperationSchedule.java

```diff
@@ -0,0 +1,54 @@
+package domain;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+public enum OperationSchedule {
+
+    MONDAY(DayOfWeek.MONDAY, LocalTime.of(8, 0), LocalTime.of(23, 0), LocalTime.of(13, 0)),
```

Comment:
```suggestion
MONDAY(DayOfWeek.MONDAY, Constants.EIGHT, Constants.TWENTY_THREE, LocalTime.of(13, 0))
```
뭔가 이렇게 공통된 부분은 상수화해두면 변경이 필요할 때 한군데만 수정해도 될 것 같은 느낌?

Reply:
현재 요구사항에서는 모든 요일에 운영 시간이 같지만 달라질 수도 있다고 생각하여 상수화하지 않았던 것 같습니다. 
반복되는 부분이 너무 많다고 생각하긴 했었는데, 상수화하니 변경에 더 유리할 것 같습니다. 가독성도 더 좋아진 것 같네요👍